AudioBar/AudioBarTests.swift

```diff
@@ -355,6 +374,69 @@ class AudioBarTests: XCTestCase, Tests {
         expect(view?.playbackTime, "-0:40")
     }
 
+    func testPlaybackDuration1() {
```

Comment:
Why do we have 4 tests for durations 1, 20, 21 and 60 seconds? We need just two — 1 and 2 seconds.